### DISCUSSION TITLE
Use Custom  Theme  with filament

### DISCUSSION THREAD

**Discussion by starking8b** [filamentphp/filament#13380] [Help]
thanks for  amazing Package which I  think it is  so  useful  ,  but I  have vuexy  template   and I  want to  use  this  vuexy  with filament .  is that  possible ?  I  did  not find any  doc talking about integrating  template with filament

**Comment by leandrocfe** [comment]
You need to create a custom theme and edit the css
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

https://filamentphp.com/docs/3.x/support/style-customization#discovering-hook-classes

**Reply by starking8b** [reply]
thanks for reply , I  just generated New  Resources using this  command    php artisan make:filament-resource User and  it is  working but  I am not able to fine user-resources.blade.php in anywhere in the system
